### ISSUE TITLE
items will despawn instead of dropping if the dropitems list has unobtainable item ids in it

### ISSUE THREAD

**Issue by MatteSevai** [Miffyli/TerrariaCustomMediumcore#1]

**Comment by Miffyli** [comment]
Hey, I am not working on this plugin anymore, but will happily accept any updates in case somebody still finds the mod useful :). Can you give a concrete example of "unobtainable" item?

**Comment by MatteSevai** [comment]
by unobtainable I mean things that are in the code, but were never made obtainable by any normal gameplay means
https://terraria.fandom.com/wiki/Unobtainable_features

**Comment by Miffyli** [comment]
Right, sounds like some part of the code fails if it encounters an "unobtainable item id", if it affects all other items too (i.e. the plugin stops working). Unfortunately I do not have time to work on this, but will happily review a suggested change :)
